packages/server/lib/browsers/firefox.ts

```diff
@@ -561,7 +561,7 @@ export async function open (browser: Browser, url: string, options: BrowserLaunc
 
       debug('closing firefox')
 
-      originalBrowserKill.apply(browserInstance, args)
+      return originalBrowserKill.apply(browserInstance, args)
```

Comment:
Needed to return this so the overridden `kill` has the correct signature (needs a return value of `boolean`)